BlueprintUI/Tests/PerformancePlayground.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  PerformancePlayground.swift
+//  BlueprintUI-Unit-Tests
+//
+//  Created by Kyle Van Essen on 6/23/20.
+//
+
+import XCTest
+@testable import BlueprintUI
+
+
+class PerformancePlayground : XCTestCase
+{
+    override func invokeTest() {
+        // Uncomment this line to run performance metrics, eg in Instruments.app.
+        // super.invokeTest()
```

Comment:
Does this prevent these tests from running normally? Since they perform no assertions they're probably not useful in CI.

Reply:
Yeah, this cuts them off from CI